.gitignore

```diff
@@ -54,3 +54,5 @@ fabric.properties
 
 # Android studio 3.1+ serialized cache file
 .idea/caches/build_file_checksums.ser
+
+sqlite.db
```

Comment:
Prefer to call it `dashboard.sqlite.db` so that it can be recognized easily when integrated into PD.